README.md

```diff
@@ -160,6 +180,22 @@ bundle install
 buf generate
 ```
 
+### Testing
+
+```bash
+# Run tests
+bundle exec rspec
+
+# Run with coverage
+COVERAGE=1 bundle exec rspec
+
+# Run linting
+bundle exec rubocop
+
+# Security audit
+bundle exec bundle-audit
```

Comment:
Don't have much experience with ruby, but is this `bundle-audit` something built-in? Or an extra dev dependency?